app/Http/Controllers/User/ProfileController.php

```diff
@@ -14,4 +19,49 @@ public function profile()
 
         return view('user.profile', compact('user'));
     }
+
+    public function edit(User $user)
+    {
+        return view('user.edit', compact('user'));
+    }
+
+    public function editPassword(User $user)
+    {
+        return view('user.updatePassword', compact('user'));
+    }
+
+    public function update(UserProfileRequest $request, User $user)
+    {
+        $user->update($request->all());
+
+        return redirect()->route('user.profile');
+    }
+
+    public function changePassword(ChangePasswordRequest $request, User $user)
+    {
+        $user->update(['password' => Hash::make($request->password)]);
```

Comment:
```suggestion
        $user->update([
            'password' => Hash::make($request->password),
        ]);
```
viết ntn cho thoáng code nhé

Reply:
Vâng ạ